### PR TITLE
Sightline description

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fixed description

### CODE REVIEW

File: Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.yml [hunk 1]
```diff
@@ -614,7 +614,7 @@ script:
           description: Relationships of the mitigation (only visible when extending the data).
           type: Unknown
       deprecated: false
-      description: List all alerts. When an alert ID is given, only the relevant alert will be fetched.
+      description: Add a TMS or flowspec mitigation with the attributes and relationships passed in the JSON sub_object.
```
Comment (Itay4) [inline Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.yml:617]:
```suggestion
      description: Add a TMS or flowspec mitigation with the attributes and relationships passed in the JSON subobject.
```

File: Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/README.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ List all mitigations. When a mitigation ID is given, only the relevant mitigatio
 
 ### na-sightline-mitigation-create
 ***
-List all alerts. When an alert ID is given, only the relevant alert will be fetched.
+Add a TMS or flowspec mitigation with the attributes and relationships passed in the JSON sub_object.
```
Comment (Itay4) [inline Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/README.md:294]:
```suggestion
Add a TMS or flowspec mitigation with the attributes and relationships passed in the JSON subobject.
```

**Review by Itay4 — APPROVED** [review]

File: Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.yml (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.yml:617]: The argument is called sub_object

Comment (ShahafBenYakir) [reply Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.yml:617]: @Itay4

Comment (Itay4) [reply Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.yml:617]: ok

**Pull request merged**